### PR TITLE
Jobs and transactions

### PR DESCRIPTION
**Pull request by rvolosatovs**
This seems to work, as the time is running out - I'd want to merge this asap and get to fixing everything based on this work(also fix this PR,if that is needed). 
Everything should be much more descriptive and the commits are not properly split, but I want to type as much code as possible, while I still _can type_

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -132,4 +132,8 @@ cleanmock:
 	@echo "Removing mock units..."
 	@-rm -rf `find $(ABS_REPO) -name 'mock_*'`
 
+fix:
+	@echo "rvolosatovs -> rvolosatovs"
```
Comment (bpowers) [inline Makefile]: probably don't need this anymore

File: fix-import.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+substitution='s:b1101:rvolosatovs:'
```
Comment (bpowers) [inline fix-import.sh]: good to leave in history, but can be removed from tree I think

File: system/daemon.go [hunk 3]
```diff
@@ -82,176 +102,246 @@ func Supported(filename string) bool {
...
-// If error is returned, it is going to be ErrNotFound
-func (sys *Daemon) IsActive(name string) (st unit.Activation, err error) {
-	var u *Unit
-	if u, err = sys.Get(name); err == nil {
-		st = u.Active()
+// Units returns a slice of all units created
+func (sys *Daemon) Units() (units []*Unit) {
+	unitSet := map[*Unit]struct{}{}
+	for _, u := range sys.units {
+		unitSet[u] = struct{}{}
```
Comment (bpowers) [inline system/daemon.go]: are there duplicate units in `sys.units`?  or is this because there can be multiple names for a single unit?

**Comment by bpowers** [comment]
I have just a few comments above.  I think it looks pretty good!

File: system/daemon.go (hunk 3 above)
Comment (rvolosatovs) [inline system/daemon.go]: as dependencies are specified using both filenames and paths(when using symlinks in a dir), each unit is stored under either path only or both path and name keys

File: Makefile (hunk 1 above)
Comment (rvolosatovs) [inline Makefile]: initially left in case something went wrong etc, not needed indeed, will remove

File: system/daemon.go (hunk 3 above)
Comment (bpowers) [inline system/daemon.go]: makes sense

**Pull request merged**